hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/s3guard/MetadataStore.java

```diff
@@ -50,17 +50,21 @@
    * Performs one-time initialization of the metadata store.
    *
    * @param fs {@code FileSystem} associated with the MetadataStore
+   * @param ttlTimeProvider
    * @throws IOException if there is an error
    */
-  void initialize(FileSystem fs) throws IOException;
+  void initialize(FileSystem fs, ITtlTimeProvider ttlTimeProvider)
```

Comment:
See discussion in DynamoDBMetadataStore.initialize() about why we don't need an explicit entry here